app/Http/Controllers/Api/ExperienceSkillsController.php

```diff
@@ -19,6 +19,17 @@ public function store(StoreExperienceSkill $request)
             ['experience_id', $data['experience_id']],
             ['experience_type', $data['experience_type']]
         ])->first();
+
+        // Attach skill to applicant if not already attached.
+        $applicant = $request->user()->applicant;
+        $skillApplicantRelationshipExists = $applicant->skills()
+            ->wherePivot('applicant_id', $applicant->id)
+            ->wherePivot('skill_id', $data['skill_id'])
+            ->exists();
```

Comment:
This seems overly verbose. Would this work equally well?
`$skillApplicantRelationshipExists = $applicant->skills->where('id', $data['skill_id'])->exists();`